docker-compose.mender-gateway.commercial.demo.yml

```diff
@@ -0,0 +1,30 @@
+version: '2.1'
+services:
+
+    #
+    # mender-gateway
+    #
+    mender-gateway:
+        networks:
+            mender:
+            mender_local:
+                aliases:
+                    - gateway.docker.mender.io
+
+    #
+    # mender-client
+    #
+    mender-client:
+        image: mendersoftware/mender-client-qemu:mender-master
+        networks:
+            mender_local:
+        stdin_open: true
+        tty: true
+        privileged: true
```

Comment:
Q:
Why privileged?

Reply:
I honestly don't know. But it is [already in the base image](https://github.com/mendersoftware/integration/blob/master/docker-compose.client.yml#L13).